source/index.ts

```diff
@@ -1,28 +1,26 @@
-/* eslint-disable node/no-deprecated-api */
-'use strict';
-const fs = require('fs');
-const path = require('path');
-const crypto = require('crypto');
-const assert = require('assert');
-const EventEmitter = require('events');
-const dotProp = require('dot-prop');
-const makeDir = require('make-dir');
-const pkgUp = require('pkg-up');
-const envPaths = require('env-paths');
-const writeFileAtomic = require('write-file-atomic');
-const Ajv = require('ajv');
-const debounceFn = require('debounce-fn');
-const semver = require('semver');
-const onetime = require('onetime');
-
-const plainObject = () => Object.create(null);
+import fs = require('fs');
+import path = require('path');
+import crypto = require('crypto');
+import assert = require('assert');
+import EventEmitter = require('events');
+import dotProp = require('dot-prop');
+import makeDir = require('make-dir');
+import pkgUp = require('pkg-up');
+import envPaths = require('env-paths');
+import writeFileAtomic = require('write-file-atomic');
+import Ajv = require('ajv');
+import debounceFn = require('debounce-fn');
+import semver = require('semver');
+import onetime = require('onetime');
```

Comment:
Is this syntax correct? Shouldn't it be `import fs from 'fs'` and the like?

Reply:
Although it does work, it's not the correct way.

Thanks for noticing this 😄

Reply:
No, this syntax is actually correct.

Reply:
2

Reply:
Updated 👍

Reply:
Oh, I finally [figured it out](https://github.com/sindresorhus/mem/issues/31#issuecomment-475858825), amazing...

Reply:
@sindresorhus Nowadays do you still recommend the `import foo = require('foo')` syntax for importing CJS-only modules?

Sorry for the clutter, but this was the "less worse" place to ask that I could think of. Thank you.